src/createServer.js

```diff
@@ -1,10 +1,31 @@
 /* eslint-disable no-console */
 'use strict';
 
-function createServer() {
-  /* Write your code here */
-  // Return instance of http.Server class
-}
+const http = require('http');
+
+const PORT = process.env.PORT || 5701;
+const baseUrl = `http://localhost:${PORT}`;
+
+const createServer = () => {
+  return http.createServer((req, res) => {
+    const normalizedURL = new URL(req.url, baseUrl);
```

Comment:
The `URL` constructor is used here with `req.url` and `baseUrl`. This works correctly for local development, but be aware that `req.url` should be a valid path. If the server is used in a different context, ensure that `req.url` is always a valid URL path.